src/exercise/client/src/App.css

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
```

Comment:
small ui feedback, always check your app on several screen sizes,
add min-width to handle that :) 
![image](https://user-images.githubusercontent.com/65164924/177036826-92f7f340-bf11-468d-93f0-513adb67cdde.png)